aws/alarms/lambda/notify_slack/notify_slack.js

```diff
@@ -23,7 +23,7 @@ function getMessage(message) {
 
 function getSNSMessageSeverity(message) {
   const errorMessages = ["Error", "Critical"];
-  const warningMessages = ["Warning", "FAILURE"];
+  const warningMessages = ["Warning", "warning", "FAILURE"];
```

Comment:
message contains: `"AlarmActions\":[\"arn:aws:sns:ca-central-1:687401027353:alert-warning\"]` and is not being detected because indexOf is case sensitive

Reply:
At this point I wonder if we should not just make sure both strings we are comparing (at line 32) are converted to lowercase.
Then we would just need to have
```
const errorMessages = ["error", "critical"];
const warningMessages = ["warning", "failure"];
```

Reply:
I was thinking the same, but opted for the least intrusive change (i.e., minimum regression)... especially since we have a feeling this whole alarm might be useless, lol. But I agree the current approach is not solid, Ill convert to lower case and do what you suggested